roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds.yaml.j2

```diff
@@ -46,13 +46,13 @@ spec:
         - name: libvirt
           image: {{ ssp_registry | default('kubevirt') }}/{{ image_name_prefix }}{{ virt_launcher_image }}:{{ virt_launcher_tag }}
           command: ["/bin/sh","-c"]
-{% if use_kvm %}
+{% if useKVM %}
           args: ["libvirtd -d; chmod o+rw /dev/kvm; virsh domcapabilities --machine q35 --arch x86_64 --virttype kvm > /etc/kubernetes/node-feature-discovery/source.d/virsh_domcapabilities.xml; cp -r /usr/share/libvirt/cpu_map /etc/kubernetes/node-feature-discovery/source.d/"]
```

Comment:
I think in this file we need to support both ways up until the next stable version, included :\